src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -82,7 +82,9 @@ public void SetGet_Roundtrips()
         [InlineData(true)]
         public void ProxySetViaEnvironmentVariable_DefaultProxyCredentialsUsed(bool useProxy)
         {
-            bool envVarsSupported = UseManagedHandler || !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            //bool envVarsSupported = UseManagedHandler || !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+	    // TODO: Issue #25640
```

Comment:
Why are we leaving commented-out code in a PR?

Also, the tab/spacing of the TODO looks weird.

Reply:
my intention was to put it back as it was before. I can remove the commented code if that is preference.

Reply:
CoreFx coding guidelines prohibit leaving in commented-out code.  Not sure how it got in there to begin with.  But if you can clean it all up, thx.

Reply:
fixed. I'm not fan of commented code my self but I viewed this is transient.